zookeeper-server/src/main/java/org/apache/zookeeper/server/watch/WatchManagerOptimized.java

```diff
@@ -349,7 +349,7 @@ public WatchesPathReport getWatchesByPath() {
      * May cause OOM if there are lots of watches, might better to forbid
      * it in this class.
      */
-    public Map<Watcher, Set<String>> getWatcher2PathesMap() {
+    public Map<Watcher, Set<String>> getWatcher2PathsMap() {
```

Comment:
Public API changed. I wonder if it affects user compatibility.

Reply:
@tisonkun I think we can discuss this in other thread, reverted